app/views/spaces/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_for @space, html: { multipart: true } do |f| %>
+  <% if @space.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@space.errors.count, "error") %> prohibited this space from being saved:</h2>
+
+      <ul>
```

Comment:
test

Reply:
Lets remove partials until we know we need them

Reply:
can you check now